lib/beaker/dsl/install_utils.rb

```diff
@@ -1453,6 +1453,57 @@ def build_ignore_list(opts = {})
         ignore_list << '..' unless ignore_list.include? '..'
         ignore_list
       end
+
+      # This method is a fix for PUP-2365 when a windows agent cannot download
+      # modules from the forge because it doesn't have the appropriate
+      # certificates installed to validate the https connection.
+      #
+      # This method installs the certifcates required in order to that functionality
+      # to work as expected.
+      #
+      # @param [Host] host             A host object
+      #
+      def install_forge_cert(host)
+        geotrust_global_ca = <<-CERT.gsub(/^ {8}/, '')
+        -----BEGIN CERTIFICATE-----
+        MIIDVDCCAjygAwIBAgIDAjRWMA0GCSqGSIb3DQEBBQUAMEIxCzAJBgNVBAYTAlVT
+        MRYwFAYDVQQKEw1HZW9UcnVzdCBJbmMuMRswGQYDVQQDExJHZW9UcnVzdCBHbG9i
+        YWwgQ0EwHhcNMDIwNTIxMDQwMDAwWhcNMjIwNTIxMDQwMDAwWjBCMQswCQYDVQQG
+        EwJVUzEWMBQGA1UEChMNR2VvVHJ1c3QgSW5jLjEbMBkGA1UEAxMSR2VvVHJ1c3Qg
+        R2xvYmFsIENBMIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEA2swYYzD9
+        9BcjGlZ+W988bDjkcbd4kdS8odhM+KhDtgPpTSEHCIjaWC9mOSm9BXiLnTjoBbdq
+        fnGk5sRgprDvgOSJKA+eJdbtg/OtppHHmMlCGDUUna2YRpIuT8rxh0PBFpVXLVDv
+        iS2Aelet8u5fa9IAjbkU+BQVNdnARqN7csiRv8lVK83Qlz6cJmTM386DGXHKTubU
+        1XupGc1V3sjs0l44U+VcT4wt/lAjNvxm5suOpDkZALeVAjmRCw7+OC7RHQWa9k0+
+        bw8HHa8sHo9gOeL6NlMTOdReJivbPagUvTLrGAMoUgRx5aszPeE4uwc2hGKceeoW
+        MPRfwCvocWvk+QIDAQABo1MwUTAPBgNVHRMBAf8EBTADAQH/MB0GA1UdDgQWBBTA
+        ephojYn7qwVkDBF9qn1luMrMTjAfBgNVHSMEGDAWgBTAephojYn7qwVkDBF9qn1l
+        uMrMTjANBgkqhkiG9w0BAQUFAAOCAQEANeMpauUvXVSOKVCUn5kaFOSPeCpilKIn
+        Z57QzxpeR+nBsqTP3UEaBU6bS+5Kb1VSsyShNwrrZHYqLizz/Tt1kL/6cdjHPTfS
+        tQWVYrmm3ok9Nns4d0iXrKYgjy6myQzCsplFAMfOEVEiIuCl6rYVSAlk6l5PdPcF
+        PseKUgzbFbS9bZvlxrFUaKnjaZC2mqUPuLk/IH2uSrW4nOQdtqvmlKXBx4Ot2/Un
+        hw4EbNX/3aBd7YdStysVAq45pmp06drE57xNNB6pXE0zX5IJL4hmXXeXxx12E6nV
+        5fEWCRE11azbJHFwLJhWC9kXtNHjUStedejV0NxPNO3CBWaAocvmMw==
+        -----END CERTIFICATE-----
+        CERT
+
+        hosts.each do |host|
+          if host['platform'] =~ /windows/
+            install_cert_on(host, 'geotrust_global_ca', geotrust_global_ca)          
+          end
+        end
+      end
+
+      # This method will install a pem file certifcate on a windows host
+      # 
+      # @param [Host] host                 A host object
+      # @param [String] cert_name          The name of the pem file
+      # @param [String] cert               The contents of the certificate
+      #                     
+      def install_cert_on(host, cert_name, cert)
```

Comment:
This is a windows only method with a generic name.  Should be called something different and also raise error if passed a host of incorrect platform.